### PR TITLE
 Resume extraction of optical flow

### PR DESCRIPTION
**Pull request by surajkothawade**
Start extracting optical flow from the next video instead of restarting from scratch.

### CODE REVIEW

File: tools/build_of.py [hunk 1]
```diff
@@ -107,13 +110,17 @@ def run_warp_optical_flow(vid_item, dev_id=0):
...
     new_size = (args.new_width, args.new_height)
     NUM_GPU = args.num_gpu
-
     if not os.path.isdir(out_path):
         print "creating folder: "+out_path
         os.makedirs(out_path)
-
-    vid_list = glob.glob(src_path+'/*/*.'+ext)
+    print src_path
+    print ext
```
Comment (yjxiong) [inline tools/build_of.py]: Remove these prints or have better formatted logs

File: tools/build_of.py [hunk 2]
```diff
@@ -107,13 +110,17 @@ def run_warp_optical_flow(vid_item, dev_id=0):
     ext = args.ext
     new_size = (args.new_width, args.new_height)
     NUM_GPU = args.num_gpu
-
     if not os.path.isdir(out_path):
```
Comment (yjxiong) [inline tools/build_of.py]: The default behavior should be overwriting. Add an option to enable resuming.

File: tools/build_of.py [hunk 3]
```diff
@@ -107,13 +110,17 @@ def run_warp_optical_flow(vid_item, dev_id=0):
...
-
-    vid_list = glob.glob(src_path+'/*/*.'+ext)
+    print src_path
+    print ext
+    vid_list = glob.glob(src_path+'/*.'+ext)
     print len(vid_list)
+    print vid_list[1]
+    com_vid_list = os.listdir(out_path)
+    vid_list = nonintersection(vid_list, com_vid_list)
+    print "Resuming from: ", vid_list[0]
```
Comment (yjxiong) [inline tools/build_of.py]: better log the number of videos.

**Comment by surajkothawade** [comment]
> Would you please use print() to make it compatible with python3

Done.

**Comment by yjxiong** [comment]
Thanks for the contribution

**Pull request merged**